app/enterprise/changelog.md

```diff
@@ -3,6 +3,156 @@ title: Kong Gateway (Enterprise) Changelog
 no_search: true
 no_version: true
 ---
+## 2.4.0.0 (beta)
+**Release Date** 2021/04/13
+
+### Features
+
+#### Core
+- This Kong version includes relaxed version check in hybrid mode between control planes
+  and data planes, allowing data planes that are missing minor updates (up to two) to
+  still connect to the control plane. Also, data planes are allowed to have a superset
+  of plugins in addition to the control plane plugins. [6932](https://github.com/Kong/kong/pull/6932)
+  <div class="alert alert-ee blue">
+    Data planes are not allowed to connect to control planes if they are a different major version,
+    a version newer than the control plane’s version, or missing plugins from the control plane.
+  </div>
+- This Kong version allows UTF-8 characters in tags. This update expands the range of
+  accepted characters in tags from a limited set of ASCII characters to almost all of UTF-8 sequences.
+  Exceptions:
+  - ',' and '/' are reserved for filtering tags with "and" and "or" and are not allowed in tags.
+  - Non-printable ASCII (like the space character) is not allowed.
+- This Kong version supports Online Certificate Status Protocol (OCSP) responder in cluster for
+  hybrid mode control planes. This new feature can be configured in the `kong.conf` file.
+  [6887](https://github.com/Kong/kong/pull/6887)
+- In this Kong version, Postgres `ssl_version` configuration now supports 'any' and will negotiate with server.
+  This update ensures that `luasec` will negotiate the most secure protocol available if not otherwise set.
```

Comment:
```suggestion
- Postgres `ssl_version` configuration now supports `any` and will negotiate with the server.
  If `ssl_version` is not set explicitly, this update ensures that `luasec` will negotiate the most secure protocol available.
```

Is it set to `any` by default now? I _think_ that's what this is saying. If true, then I'd rephrase to something like:

- Postgres `ssl_version` configuration now defaults to `any`. If `ssl_version` is not explicitly set, the `any` option ensures that `luasec` will negotiate the most secure protocol available.